cfme/cloud/instance/__init__.py

```diff
@@ -189,7 +189,7 @@ def power_control_from_cfme(self, *args, **kwargs):
         if kwargs.get('from_details', True):
             navigate_to(self, 'Details')
         else:
-            navigate_to(self, 'ProviderAll')
+            navigate_to(self, 'AllForProvider')
```

Comment:
im starting to thing we should start to switch from strings to types for this kind of lookup, can we get a discussion on that?